cmd/machine-config-operator/bootstrap.go

```diff
@@ -36,6 +38,8 @@ var (
 
 func init() {
 	rootCmd.AddCommand(bootstrapCmd)
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.etcdCAFile, "etcd-ca", "/etc/ssl/etcd/ca.crt", "path to etcd CA certificate")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.rootCAFile, "root-ca", "/etc/ssl/kubernetes/ca.crt", "path to root CA certificate")
```

Comment:
Don't we want to look for these [under `/etc/kubernetes`][1]?

[1]: https://github.com/openshift/installer/issues/622#issuecomment-436317953

Reply:
These are default values.
Also MCO runs inside a container so volume mounts take take of most of the stuff.

Reply:
> These are default values.

Why not match the defaults to paths that are more native?  If we expect this to run from a container (and we do), then `/etc/kubernetes/...` seems like a better fit.  If we expect this to be run on the host (without a container runtime), then `/etc/pki/...` seems like a better fit.  @cgwalters has `/etc/ssl/...` as [a Debian/Gentoo-ism][1].

[1]: https://github.com/openshift/installer/issues/622#issuecomment-436318092

Reply:
> Why not match the defaults to paths that are more native? If we expect this to run from a container (and we do), then /etc/kubernetes/... seems like a better fit.

inside the container `/etc/ssl` seem more useful; `/etc/kubernetes` is not required inside the container.

gonna keep the defaults as-is unless required.

Reply:
> inside the container /etc/ssl seem more useful;

Yeah, I doubt we have a lot of consistency on this point.  Keeping `/etc/ssl` inside the container seems OK to me too.